### ISSUE TITLE
"Telemetry lost" audio delayed or missing

### ISSUE THREAD

**Issue by raphaelcoeffic** [opentx/opentx#6593]
**Version tested:** 
2.3.0N55

**Hardware:**
X9 lite with latest ISRM firmware supporting D16

**Description**
Receiver is bound using ACSST D16. When unplugging the receiver, it takes a longer time to play the "Telemetry lost" audio, and sometimes it is not played at all ("RSSI critical" is played though).

I noticed that compare to my X10 on 2.2.4, the ISRM module seems to continue sending telemetry packets for some seconds (RSSI=0db; Rx voltage=normal). The "telemetry lost" audio (if played), can be heard only after the last telemetry packet has arrived.

**Comment by raphaelcoeffic** [comment]
@3djc wrote on chat:
```yes, I just replicated with a 2.3 x7, unplug receiver, 2 or 3 secs rssi critical, 5 or more secs, telemetry lost, which wasn't 2.2 behavior I think```

**Comment by raphaelcoeffic** [comment]
duplicate of #6548
